k8s/grafana/README.md

```diff
@@ -8,6 +8,36 @@ data collection systems - including Prometheus, InfluxDB, Elasticsearch, MySQL o
 
 Popular open stacks on Kubernetes packaged by Google.
 
+## Design
+
+![Architecture diagram](resources/grafana-architecture.png)
```

Comment:
Please, add information about Grafana ports (both to the text and diagram).

Reply:
It's also not clear to me whether it is a single-node or multi-node design.

Reply:
Done.